Maple2.Server.Game/Manager/ExperienceManager.cs

```diff
@@ -103,6 +103,7 @@ public sealed class ExperienceManager {
         expGained += GetRestExp(expGained);
         LevelUp();
         AddPrestigeExp(message.Type());
+        // TODO: Achievement 22400001, 22400002
```

Comment:
isnt just the case of using `ConditionUpdate` ?

Reply:
added, please double check it though since idk what im doing

Reply:
it looks like there's neither any code or target for exp (judging based on available quests/achievements with this condition. Add these cases to return true in `ConditionUtil.CheckCode` and `CheckTarget` in the switches so we know they've been addressed on those ends.